### DISCUSSION TITLE
trento-agent and gatherer plugin

### DISCUSSION THREAD

**Discussion by angelabriel** [trento-project/wanda#118] [Q&A] [answered]
I have a VM with installed trento-agent package version 1.2.0-150300.1.11.1

rpm -qa | grep trento-agent
trento-agent-1.2.0-150300.1.11.1.x86_64

The 'corosync.conf' gatherer is working

/usr/bin/trento-agent facts gather --gatherer "corosync.conf" --argument "quorum.two_node
"
INFO[2022-12-06 18:38:12] Using config file: /etc/trento/agent.yaml    
INFO[2022-12-06 18:38:12] loading plugins                              
INFO[2022-12-06 18:38:12] Starting corosync.conf file facts gathering process 

But the 'sbd_config' or the 'systemd' gatherer is NOT working

/usr/bin/trento-agent facts gather --gatherer "sbd_config" --argument "SBD_DEVICE"
INFO[2022-12-06 18:36:36] Using config file: /etc/trento/agent.yaml    
INFO[2022-12-06 18:36:36] loading plugins                              
FATA[2022-12-06 18:36:36] gatherer sbd_config not found      

and

/usr/bin/trento-agent facts gather --gatherer "systemd" --argument "sbd"
INFO[2022-12-06 18:37:17] Using config file: /etc/trento/agent.yaml    
INFO[2022-12-06 18:37:17] loading plugins                              
FATA[2022-12-06 18:37:17] gatherer systemd not found 

Do I need a newer version of the trento-agent package (which one and where to get) or is it a configuration problem?

**Comment by fabriziosestito** [comment, marked as answer]
@angelabriel you need to install the factory package to have access to the latest gatherers:

https://build.opensuse.org/package/show/devel:sap:trento:factory/trento-agent

**Reply by angelabriel** [reply]
yeah, many thanks. Works like a charm.
